client/components/admin_form/README.md

```diff
@@ -0,0 +1,37 @@
+# Admin Form
```

Comment:
This is used to DRY up the generic admin form UI verticals and organizations share... if more of these conventional admin-only UIs appear I might even go a step further and make a configurable app that can be mounted b/c they share similar routing code too.